elasticsearch-transport/test/unit/transport_manticore_test.rb

```diff
@@ -99,12 +99,8 @@ class Elasticsearch::Transport::Transport::HTTP::ManticoreTest < Test::Unit::Tes
           }
         }
 
-        ::Manticore::Client.expects(:new).with(:options => {:ignore_ssl_validation => true})
-
+        ::Manticore::Client.expects(:new).with(options)
         transport = Manticore.new :hosts => [ { :host => 'foobar', :port => 1234 } ], :options => options
-
-        assert_equal java.lang.System.getProperty("javax.net.ssl.trustStore"), "test.jks"
```

Comment:
Can we somehow keep the tests that SSL was _actually_ set for the client with the new implementation? In the refactored test, we only check that the options hash has been passed...